lib/plugins/aws/deploy/lib/uploadArtifacts.js

```diff
@@ -167,7 +209,9 @@ module.exports = {
     );
 
     if (this.serverless.utils.fileExistsSync(artifactFilePath)) {
-      this.serverless.cli.log('Uploading custom CloudFormation resources...');
+      // TODO: ADD PROGRESS SUPPORT FOR IT
+      legacy.log('Uploading custom CloudFormation resources...');
+      progress.get('main').notice('Uploading custom CloudFormation resources');
```

Comment:
I think in this should be under generic `Uploading..`  (same as we do with CF template)

Reply:
yep, that was a leftover that was changed in previous iteration